### PR TITLE
Update PLCR to 1.5.1

### PR DESCRIPTION
**Pull request by Jamminroot**
Things to consider before you submit the PR:

* [ ] Has `CHANGELOG.md` been updated?
* [x] Are tests passing locally?
* [x] Are the files formatted correctly?
* [ ] Did you add unit tests?
* [ ] Did you check UI tests on the sample app? They are not executed on CI.
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

This PR updates PLCR binaries and headers used in SDK.

## Related PRs or issues

[AB#77610](https://msmobilecenter.visualstudio.com/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/77610)

### CODE REVIEW

**Comment by iamclement** [comment]
Can you please update the changelog?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,19 @@
 # App Center SDK for iOS, macOS and tvOS Change Log
 
+## Version 3.2.1 (In development)
```
Comment (MatkovIvan) [inline CHANGELOG.md]:
```suggestion
## Version 3.1.1 (In development)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,11 +1,19 @@
...
+* **[Improvement]** Update PLCrashReporter to 1.5.1
+___
+
 ## Version 3.1.0
 
 ### App Center Distribute
 
 * **[Feature]** Add a `disableAutomaticCheckForUpdate` API that needs to be called before SDK start in order to turn off automatic check for update. 
 * **[Feature]** Add a `checkForUpdate` API to manually check for update.
+___
```
Comment (MatkovIvan) [inline CHANGELOG.md]:
```suggestion

___
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,11 +1,19 @@
 # App Center SDK for iOS, macOS and tvOS Change Log
 
+## Version 3.2.1 (In development)
+
+### App Center Crashes
+
+* **[Improvement]** Update PLCrashReporter to 1.5.1
+___
```
Comment (MatkovIvan) [inline CHANGELOG.md]:
```suggestion

___
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,11 +1,19 @@
 # App Center SDK for iOS, macOS and tvOS Change Log
 
+## Version 3.2.1 (In development)
+
+### App Center Crashes
+
+* **[Improvement]** Update PLCrashReporter to 1.5.1
```
Comment (MatkovIvan) [inline CHANGELOG.md]:
```suggestion
* **[Improvement]** Update PLCrashReporter to 1.5.1.
```

**Pull request merged**